src/utils.cpp

```diff
@@ -443,24 +430,38 @@ void getVpdDataInVector(const std::string& vpdFilePath,
     }
 }
 
-size_t getVPDOffset(const nlohmann::json& parsedJson,
+size_t getVPDOffset(const nlohmann::json& i_jsonObj,
```

Comment:
I don't see any need to changing this API. All the newly added code needs to be handled at caller's end.

Reply:
If the redundant EEPROM path format is similar to what we have currently in JSON then this logic need not be handled by this API. The logic of extracting the primary path should reside with caller.
And, if there is any change in the format, please push the updated JSON for reference.